package.json

```diff
@@ -12,21 +12,22 @@
     "yarn": ">=1.0.0"
   },
   "dependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/plugin-proposal-class-properties": "^7.3.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.3.4",
+    "@babel/core": "^7.4.0",
+    "@babel/plugin-proposal-class-properties": "^7.4.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.0",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/plugin-transform-destructuring": "^7.3.2",
-    "@babel/plugin-transform-regenerator": "^7.3.4",
-    "@babel/plugin-transform-runtime": "^7.3.4",
-    "@babel/polyfill": "^7.2.5",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/runtime": "^7.3.4",
+    "@babel/plugin-transform-destructuring": "^7.4.0",
+    "@babel/plugin-transform-regenerator": "^7.4.0",
+    "@babel/plugin-transform-runtime": "^7.4.0",
+    "@babel/polyfill": "^7.4.0",
```

Comment:
sorry, needed what? `@babel/polyfill`? yes, if we remove it, then it is going to be a breaking change

Reply:
(Disclaimer: I know Babel and core-js pretty well, but not webpacker)

Why would it be a breaking change? Is it exposed to your users or is it only added by webpacker?

Reply:
![image](https://user-images.githubusercontent.com/16782063/56233814-8878ad80-6049-11e9-8ca3-7ecd7858aaca.png)
@somebody32 The document you linked seems to indicate that it needs to be removed. Apart from that, I feel like this is defenitly a breaking change since users will need to change their pack files.

Reply:
@nicolo-ribaudo @jakeNiemiec you're right, I for some reason thought that core-js@3 and @babel/polyfill could coexist for some time, but looks like it is not the cause. My idea initially was not to remove @babel/polyfill, so builds which indirectly dependant on it will not be broken, but looks like it is safer to remove it so there will be no strange compatibility problems. I'll change that

Reply:
Done!